active/0000-firefox-in-ci.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2016-12-04
+- RFC PR: 
+- Ember CLI Issue: 
+
+# Summary
+
+Replace PhantomJS with Firefox as the default browser for continuous integration testing.
```

Comment:
should we consider chrome as a default (and make it easy to run FF etc secondary), as it is most likely to mimic the developers setup?

using chrome http://blog.500tech.com/setting-up-travis-ci-to-run-tests-on-latest-google-chrome-version/